lexicon/providers/dreamhost.py

```diff
@@ -0,0 +1,181 @@
+"""Module provider for Dreamhost"""
+from __future__ import absolute_import
+import json
+import logging
+import time
+
+import requests
+from lexicon.providers.base import Provider as BaseProvider
+
+
+LOGGER = logging.getLogger(__name__)
+
+NAMESERVER_DOMAINS = ['dreamhost.com']
+
+
+def provider_parser(subparser):
+    """Module provider for Linode"""
+    subparser.add_argument(
+        "--auth-token", help="specify api key for authentication")
+
+
+class Provider(BaseProvider):
+    """Provider class for Linode"""
```

Comment:
You should replace Linode for Dreamhost here.

Reply:
I'll replace to Dreamhost.